content/docs/sidebar.json

```diff
@@ -215,6 +215,17 @@
         "label": "dag",
         "slug": "dag"
       },
+      {
+        "label": "data",
+        "slug": "data",
+        "source": "data/index.md",
```

Comment:
Can be reduced 🙂 
```suggestion
        "slug": "data",
```

Reply:
It looks like we need `label`, so I added it back. Otherwise it shows up capitalized.